### PR TITLE
#TRABootCamp-95  :[TDD] Implement a method that can split any string using a delimiter.

### PR DESCRIPTION
**Pull request by iamabdh**
https://trello.com/c/KNzZD3BW

### CODE REVIEW

**Review by BaraahAlKalbani — APPROVED** [review]
Great Work 👍🏻

**Review by AnwaarAlbusaidi — APPROVED** [review]
Done

File: src/main/java/io/code/CodeExample.java [hunk 1]
```diff
@@ -1,12 +1,28 @@
 package io.code;
 
+import java.util.ArrayList;
+
 public class CodeExample {
 	
 	public static void main(String[] arg) {
```
Comment (CodelineAtyab) [inline src/main/java/io/code/CodeExample.java:19]: Is there a need for main method, when we already have unit tests to test the implementation ?

**Review by CodelineAtyab — APPROVED** [review]

File: src/main/java/io/code/CodeExample.java (hunk 1 above)
Comment (iamabdh) [reply src/main/java/io/code/CodeExample.java:19]: Sometime unit test errors not helpful so i rely on main method to print to see what goes wrong.

**Pull request merged**